### PR TITLE
Adjust naming and description of canned reports

### PR DESCRIPTION
**Pull request by kjain110**
https://issues.folio.org/browse/MODLISTS-47

### CODE REVIEW

File: src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+UPDATE list_details SET name = 'Missing items', description = 'Returns all items with a status of: missing, aged to list, claimed returned, declared lost, long missing'
```
Comment (bvsharp) [inline src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql:1]: 'aged to list' should be 'aged to lost'

File: src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql
Comment (mweaver-ebsco) [inline src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql:1]: Not a big deal (and I definitely wouldn't bother updating the PR), but for tiny little changes like this where it's just a couple lines of SQL, I'd just put the change in a `<sql>` block in the changeset

File: src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <include file="sql/update-list-details-table.sql" relativeToChangelogFile="true"/>
```
Comment (mweaver-ebsco) [inline src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml:7]: I don't think you can use `<include>` with a SQL file. I think it needs to be a `<sqlFile>` inside a `<changeSet>` block.

Comment (kjain110) [reply src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml:7]: @mweaver-ebsco, here I have created another folder called sql inside v1.0.2 to include all the sql files in one folder. And I am just including that file with the path through `<include>`. This is the same thing we had for earlier version.

File: src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql
Comment (kjain110) [reply src/main/resources/db/changelog/changes/v1.0.2/sql/update-list-details-table.sql:1]: @mweaver-ebsco, Will make sure to implement this from next time.

File: src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml (hunk 2 above)
Comment (mweaver-ebsco) [reply src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml:7]: Huh... I had no idea that that worked haha

**Review by mweaver-ebsco — APPROVED** [review]

**Pull request merged**